gui/components/swipe/InterestsTable.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { FlatList, StyleSheet, Text, View } from "react-native";
+
+const Item = ({ item }) => (
+  <View style={styles.item}>
+    <Text>{item.activity}</Text>
+    <Text>{item.attitude}</Text>
+    <Text>{item.skillLevel}</Text>
+  </View>
+);
```

Comment:
`Item` should be it's own component. Anything that could be used somewhere else or in a loop should be its own component in its own file.